doc/userGuide.md

```diff
@@ -309,3 +309,72 @@ openjdk-tests/test-results/openjdk/report
 ```
 
 The JTREG report HTML summary file is then located at `openjdk-tests/test-results/openjdk/report/html/index.html`
+
+## Disable test
+
+#### Automatically exclude a test target
+In the issue that describes the test failure, add a comment with the following format:
```

Comment:
change to
Instead of having to manually create a PR to disable test targets, they can now be automatically disabled via Github workflow (see [autoTestPR.yml](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/.github/workflows/autoTestPR.yml)).  In the issue that describes the test failure, add a comment with the following format: